packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -206,15 +213,57 @@ void diagnoseXcodeBuildFailure(XcodeBuildResult result) {
     printError("  open ios/Runner.xcodeproj");
     return;
   }
+  if (result.buildForDevice
+      && result.buildCommands != null
```

Comment:
if you put the && at the end of the lines you'll get the subexpressions lined up